include/asterisk/channel.h

```diff
@@ -5084,5 +5113,20 @@ void *ast_channel_get_stream_topology_change_source(struct ast_channel *chan);
 #define ast_channel_has_tech_function(chan, function) \
 	(ast_channel_tech(chan) ? ast_channel_tech(chan)->function != NULL : 0)
 
+/*!
+ * \brief Get the name of the current channel storage driver
+ *
+ * \return The name of the current channel storage driver
+ */
+const char *ast_channel_get_current_storage_driver_name(void);
+
+/*!
+ * \brief Set the current channel storage driver
```

Comment:
Document what happens if this is called multiple times

Reply:
Now marked as internal and will return an error if called after ast_fdully_booted.  Documented appropriately.